docs/download.html

```diff
@@ -0,0 +1,520 @@
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Introduction to the Administration Portal</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Introduction to the VM Portal</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Self-Hosted Engine Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Data Warehouse Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store User Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Disaster Recovery Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Gluster Hyperconvergence Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Orb</a></li>
+        <li class="ov-nav-main__sub-item ov-nav-main__sub-item--heading">Release notes</li>
+        <li class="ov-nav-main__sub-item">Latest Release Notes</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Data Warehouse Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+        <li class="ov-nav-main__sub-item">Older Release Notes</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Data Warehouse Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+      </ul>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="developers.html">Developers</a>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="community.html">Community</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="https://lists.ovirt.org/archives/">Forum</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="blog.html">Blog</a></li>
+  </ul>
+</nav>
+  </header>
+  <main class="ov-main ov-main--home">
+    <section class="ov-masthead ov-masthead--download">
+      <h1 class="ov-heading ov-heading--xl">Download oVirt</h1>
+      <h2 class="ov-heading ov-heading--lg">The preferred way to install oVirt is by downloading the Self-Hosted Engine.</h2>
+      <a href="" class="ov-button ov-button--primary">Install Self-Hosted Engine</a>
+    </section>
+
+    <section class="ov-layout-styles ov-content-styles">
+      <h2 class="ov-heading ov-heading--lg">oVirt Quickstart Guide</h2>
```

Comment:
you might want to create a heading style rather than use this ov-heading styles.

Reply:
Can do! I can make a specific style for it.